src/app-layer-dhcp.c

```diff
@@ -27,7 +27,7 @@
 #include "app-layer-dhcp.h"
 
 #ifdef HAVE_RUST
-#include "rust-dhcp-dhcp-gen.h"
+#include "rust-bindings.h"
```

Comment:
Instead of all these includes, can we simply include "rust-bindings.h" in suricata-common.h? Or some other central location(s)?

Reply:
I'd prefer that approach as well. Was just trying to keep from adding additional includes, since the original approach had it with the file using it.